src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -289,6 +289,37 @@
  * }
  * </pre>
  * <p>
+ * To configure the check so that it limit IllegalType to validate field types only:
```

Comment:
I would make this a bit more concise: "To configure the check to target fields only:"

Reply:
done

Reply:
Resolved.